package.json

```diff
@@ -60,7 +60,7 @@
     "jest-watch-typeahead": "^0.6.0",
     "jquery": "^3.2.1",
     "lerna": "^4.0.0",
-    "micromatch": "^4.0.2",
+    "micromatch": "^4.0.4",
```

Comment:
this doesn't really affect much outside of this repo, but doesn't hurt either. But perhaps we could bump it in all the places we use it in the published packages?

```sh-session
$ rg '"micromatch"' -tjson
package.json
63:    "micromatch": "^4.0.2",

packages/jest-config/package.json
42:    "micromatch": "^4.0.2",

packages/jest-message-util/package.json
25:    "micromatch": "^4.0.2",

packages/jest-util/package.json
22:    "micromatch": "^4.0.2"

packages/jest-core/package.json
36:    "micromatch": "^4.0.2",

packages/jest-haste-map/package.json
27:    "micromatch": "^4.0.2",

packages/jest-transform/package.json
27:    "micromatch": "^4.0.2",
```

Reply:
This one is needed beacuse otherwise the new tests fail, because micromatch itself had a bug with negated extglobs.
Will update other locations and dedup.